pkg/ccl/backupccl/backup_test.go

```diff
@@ -6239,6 +6239,9 @@ func TestBackupRestoreTenant(t *testing.T) {
 
 	t.Run("inside-tenant", func(t *testing.T) {
 		tenant10.Exec(t, `BACKUP DATABASE foo TO 'userfile://defaultdb.myfililes/test'`)
+		tenant10.Exec(t, `CREATE DATABASE foo2`)
+		tenant10.Exec(t, `RESTORE foo.bar FROM 'userfile://defaultdb.myfililes/test' WITH into_db='foo2'`)
```

Comment:
I think we should restore into tenant11, instead of 10, so we actually have a different tenant prefix?

Slightly annoying since I think that means we can't use userfile, and I'm pretty sure node local isn't allowed in tenants either.

Reply:
http_storage_test.go:54 might be useful?